stake-accounts/src/args.rs

```diff
@@ -76,6 +79,7 @@ pub(crate) enum Command<P, K> {
 pub(crate) struct Args<P, K> {
     pub config_file: String,
     pub url: Option<String>,
+    pub commitment: Option<String>,
```

Comment:
nit: Since `no_wait` is a global arg, how about adding it here instead of the individual structs?

Reply:
Ah that makes sense, thanks!